examples/transmissions/schedule_transmission.py

```diff
@@ -2,8 +2,12 @@
 
 sp = SparkPost()
 
-response = sp.transmissions.send(
-    recipients=[
+response = sp.transmissions.post({
+    'options': {
+        'sandbox': True,
+        'start_time': '2015-11-06T09:10:00-05:00',
```

Comment:
Can we show how to relate python datetime to our start_time format? It's a really common question. Something like this?

(Jeez python - include some tzinfo batteries):

```python
from datetime import datetime, timedelta, tzinfo

class EasternTime(tzinfo):
    """UTC+5 tzinfo implementation"""
    # can be configured here
    _offset = timedelta(hours = -5)
    _dst = timedelta(hours=1)
    _name = "-05:00"
    def utcoffset(self, dt):
        return self.__class__._offset
    def dst(self, dt):
        return self.__class__._dst
    def tzname(self, dt):
        return self.__class__._name

py_start_time = datetime(2015, 1, 6, 9, 10, 0, tzinfo=EasternTime())
start_time = py_start_time.strftime('%Y-%m-%dT%H:%M:%S%Z')
```